android/app/build.gradle

```diff
@@ -33,7 +33,7 @@ def getKeystoreProperties(fileName) {
 }
 
 android {
-    compileSdkVersion flutter.compileSdkVersion
+    compileSdkVersion 34
```

Comment:
Tady si nejsem jistý, ale nepřišel jsem na to jak a kde nastavit tuto hodnotu do `flutter.compileSdkVersion` 🤔